config.py

```diff
@@ -34,7 +34,7 @@
 
 DSP_API_URL = os.environ.get('DSP_API_URL', os.environ.get('USI_API_URL', 'https://submission-test.ebi.ac.uk'))
 
-JSON_DIR = os.path.dirname(__file__) + '/tests/json/'
+JSON_DIR = os.path.dirname(__file__) + '/tests/unit/json/'
```

Comment:
What is the purpose of the 'unit' folder?
To separate unit tests and integration tests?

Reply:
yep, as in any our repos